suite/tests/client-interface/memory_dump_test.templatex

```diff
@@ -0,0 +1,40 @@
+thank you for testing memory dump
+thread init
+nudge delivered 1
+Hello, world!
+#ifdef LINUX
+ELF Header:
+  Magic:   7f 45 4c 46 02 01 01 03 00 00 00 00 00 00 00 00.*
+  Class:                             ELF64
+  Data:                              2's complement, little endian
+  Version:                           1 \(current\)
+  OS\/ABI:                            \U\N\I\X - GNU
```

Comment:
Not understanding why these escapes are here on any of these characters (/, U, N, I, X)?  They should all be removed, right?

Reply:
"UNIX" is replaced with "1" without the escape characters, so the escape characters are needed.

Reply:
Seems clearer to add `#undef UNIX` right here?  (If left this way please add a comment explaining for future readers.)

What ab the `/`?

Reply:
Good point. I've changed the code to use #undef UNIX instead of escape characters. 

Also fixed the /.